vcd/resource_vcd_network_isolated.go

```diff
@@ -315,11 +333,22 @@ func genericVcdNetworkIsolatedRead(d *schema.ResourceData, meta interface{}, ori
 		}
 		err := d.Set("dhcp_pool", newSet.List())
 		if err != nil {
-			return fmt.Errorf("[isolated network read] dhcp set %s", err)
+			return diag.Errorf("[isolated network read] dhcp set %s", err)
 		}
 	}
 	dSet(d, "description", network.OrgVDCNetwork.Description)
 
+	metadata, err := network.GetMetadata()
+	if err != nil {
+		log.Printf("[DEBUG] Unable to find network metadata: %s", err)
```

Comment:
Could you check all logs and error messages in this PR and add exact resource, not generic "something with network" -> better "something with isolated network". Thank you

Reply:
Changed, thanks! (Except for function `createOrUpdateNetworkMetadata` which is shared among the three types of networks)